### ISSUE TITLE
Remove usage of obsoleted LanguageManager

### ISSUE THREAD

**Issue by bjuris** [BVNetwork/404handler#12]
The views are using LanguageManager.Instance.Translate, LanguageManager has been obsoleted some while ago and is being removed in CMS 10.0.0. 

Replace usage with LocalizationService.Current.GetString and replace imported namespace from EPiServer.Core to EPiServer.Framework.Localization.

**Comment by evest** [comment]
Released in 4.2.1

**Comment by tobnyl** [comment]
Hi! I'm using EPiServer 10.6.0.0 and 404Handler 10.1.0. I can't use 404Handler anymore because LanguageManager has been removed in EPiServer.

Wasn't this resolved in release 4.2.1?

```
Compiler Error Message: CS0103: The name 'LanguageManager' does not exist in the current context
<%=LanguageManager.Instance.Translate("/gadget/redirects/oldurl")%>

Source File: /EPiServer/BVNetwork.404Handler/Views/NotFoundRedirect/Redirects.ascx    Line: 36 
```
